### DISCUSSION TITLE
Table does not exist in the current database error in new production MySQL database

### DISCUSSION THREAD

**Discussion by mikayelgr** [prisma/prisma#7500] [Q&A] [answered]
Hello, I am having this issue that when I deploy my app to Heroku and then link a new database to it I get this error:

```
(node:21) UnhandledPromiseRejectionWarning: Error: The table `post` does not exist in the current database.
```

Running `npx prisma migrate dev` on my local development environment fixed that but I don't have a clue on how to do the same thing in production on Heroku on a new database.

Is there any way to get through this?

**Comment by janpio** [comment]
How did you get to this situation? 
If it is a new database, you generally have to run Migrations on it e.g. via `npx prisma migrate deploy`.

**Reply by mikayelgr** [reply]
I have deployed the app to Heroku that has a brand new database without running migrations. I have now included the migration script into my `package.json` start script though I don't know whether it's the right way of doing it or not.

Here's the updated version of my `package.json`:

```json
...
"scripts": {
    "start": "npx prisma migrate deploy & node src/index"
}
...
```

**Reply by ryands17** [reply]
The script should be:

```json
"scripts": {
    "start": "npx prisma migrate deploy && node src/index"
}
```

The single `&` will make those run in parallel which is most probably why the table doesn't exist as `migrate` has not completed its operation yet.

**Reply by mikayelgr** [reply]
My script worked fine. The problem was that I couldn’t understand how can I execute a migration on Heroku. 
My initial script didn’t include the migration command that’s why it wasn’t working, but thanks for trying to help out

**Comment by mikayelgr** [comment, marked as answer]
Final solution for those who are confused

Just replace your start script with this

```json
"scripts": {
    "start": "npx prisma migrate deploy && node src/index"
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
